jdaviz/configs/cubeviz/plugins/spectral_extraction/tests/test_spectral_extraction.py

```diff
@@ -217,17 +217,17 @@ def test_aperture_markers(cubeviz_helper, spectrum1d_cube):
         assert mark.x[1] == before_x[1]
 
         slice_plg.slice = 0
-        extract_plg._obj.dev_cone_support = True
-        extract_plg._obj.wavelength_dependent = True
-        assert mark.x[1] == before_x[1]
+        with cubeviz_helper.app._ff_temporarily_enabled('cone_apertures'):
```

Comment:
and lastly, tests that use the feature can use this context manager to temporarily enable them for testing.  Once the feature is ready to be made public, we can either set the default to True, or preferably just search for all instances of the label, remove those lines, and remove some indentation.